django_netjsongraph/base/admin.py

```diff
@@ -60,7 +74,7 @@ def get_actions(self, request):
         return actions
 
     def get_extra_context(self, pk=None):
-        ctx = {}
+        ctx = super(AbstractTopologyAdmin, self).get_extra_context(pk)
```

Comment:
unfortunately the `submit_line` value needs to be hardcoded (`admin/django_netjsongraph/submit_line.html`) because otherwise `openwisp_network_topology` will try to look in `admin/openwisp_network_topology/submit_line.html` and fail. I have just tested this in my local dev env